app/controllers/util/CacheIdGenerator.scala

```diff
@@ -19,11 +19,15 @@ import models.requests.AuthenticatedRequest
 
 object CacheIdGenerator {
 
+  val itemsId = "exportItems"
```

Comment:
Please move it out of `CacheIdGenerator`. It's generator, not a place for all `IDs` I think better to put it inside the controller/model.